tasks/reputationManagement.js

```diff
@@ -4,17 +4,6 @@ const initialRepBalances = require("../inputs/initialRepBalances.json");
 
 const { formatEther } = utils;
 
-task("verifyReputation", "Verifies the contract on etherscan").setAction(
```

Comment:
For my own understanding, why did you take this part out but not replacing it with something else? Was this only for testing?

Reply:
I believe the information is in the issue :) 
https://github.com/PrimeDAO/contracts-v2/issues/107

Reply:
lmk if it is still unclear

Reply:
You're right, it cleared things up. I didn't think about looking into the issue. Thanks